lib/some.ml

```diff
@@ -429,7 +429,7 @@ let print_branch conf base psn name =
         (_, _, select) :: _ -> select
       | _ -> None
     in
-    let print_elem p with_link with_id =
+    let print_elem p with_link with_id with_sn =
```

Comment:
What is the difference between `with_sn` and `psn`? I guess that `psn` stand for "print surname", so I do not think there is a need for a new argument here.

Reply:
probable, mais il vaudrait mieux valider avec @lledieu

Reply:
D'après le commentaire du commit, j'ai été obligé de rajouter ça pour gérer le cas de figure où le nom des 2 époux est le même ! psn est un paramètre utilisateur pour conditionner l'affichage, with_sn est un paramètre technique pour forcer l'affichage du nom quand on affiche l'époux... Donc ce n'est pas la même chose.

Reply:
D'après le code `print_elem p true true false`  ça semble codé en dur et pas être un paramètre utilisateur ?

Reply:
Au temps pour moi j'ai vu 3 boolean a chaque fois et j'ai cru qu'il y en avais un pour psn mais non